### PR TITLE
Fix the error when there are no IDPs configured.

### PR DESCRIPTION
**Pull request by HimashiRathnayake**
## Purpose
This PR fixes the following error when there are no IDPs configured.

Screenshot_2021-03-21 Authentication Flow Composer

### CODE REVIEW

File: src/components/core/header/app-header.tsx [hunk 1]
```diff
@@ -51,7 +52,9 @@ export const AppHeader: FunctionComponent = () : ReactElement => {
             });
         getAuthenticators("idp")
             .then((response) => {
-                setAuthFactors((elements: any)=>[...elements, ...response.data.identityProviders]);
+                if (response.data.identityProviders) {
```
Comment (brionmario) [inline src/components/core/header/app-header.tsx:55]: What should be the value of `authFactors` if `response.data.identityProviders` is not defined?

File: src/utils/mapper/code-generator.ts [hunk 2]
```diff
@@ -19,6 +19,125 @@
...
 import { File } from "@babel/types";
 
+/**
+ * Generate code from and AST(Abstract Syntax Tree).
+ *
+ * {@link https://babeljs.io/docs/en/babel-generator}
+ *
+ * @example
+ * // returns "var a = 2"
+ * generateCodeFromAst({
```
Comment (brionmario) [inline src/utils/mapper/code-generator.ts:29]: Shall we shorten the example.

```suggestion
 * generateCodeFromAst({
 *         "type": "File",
 *          ....
 * });
```

File: src/utils/mapper/parser.ts [hunk 3]
```diff
@@ -19,6 +19,125 @@
...
 import { File } from "@babel/types";
 
+/**
+ * Generate code from and AST(Abstract Syntax Tree).
+ *
+ * {@link https://babeljs.io/docs/en/babel-parser}
+ *
+ * @example
+ * // returns {
+    "type": "File",
```
Comment (brionmario) [inline src/utils/mapper/parser.ts]: Let's shorten this example.
```suggestion
    "type": "File",
    ...
 }
```

**Review by brionmario — APPROVED** [review]

File: src/utils/mapper/code-generator.ts (hunk 2 above)
Comment (HimashiRathnayake) [reply src/utils/mapper/code-generator.ts:29]: Fixed with [2c1ee22](https://github.com/wso2-incubator/identity-authentication-flow-composer/pull/20/commits/2c1ee2255be1a957456f703300dfac35707c0710)

File: src/utils/mapper/parser.ts (hunk 3 above)
Comment (HimashiRathnayake) [reply src/utils/mapper/parser.ts]: Fixed with [322df22](https://github.com/wso2-incubator/identity-authentication-flow-composer/pull/20/commits/322df22bc4c2dbe9e50a2be465afedde8bde3b3b)

File: src/components/core/header/app-header.tsx (hunk 1 above)
Comment (HimashiRathnayake) [reply src/components/core/header/app-header.tsx:55]: Fixed with [a68afe2](https://github.com/wso2-incubator/identity-authentication-flow-composer/pull/20/commits/a68afe278686c3df15a57d4748efed6b1fe75f06)

**Review by brionmario — APPROVED** [review]

**Pull request merged**